core/carp_float.h

```diff
@@ -21,6 +21,12 @@ float Float_from_MINUS_int(int x) {
     return (float)x;
 }
 
+int Float_to_MINUS_bytes(float x) {
+    int y;
```

Comment:
this should also probably be unsigned

Reply:
I don’t think that’s necessary, and we don’t actually have unsigned types in the core library yet.